readthedocs/core/static-src/core/js/doc-embed/version-compare.js

```diff
@@ -6,17 +6,22 @@ function init(data) {
 
     /// Out of date message
 
-    if (data.is_highest) {
+    if (data.is_stable) {
         return;
     }
 
+    var message = 'You are not using the most up to date version of the library.';
+    if (data.is_highest) {
+      message = 'You are not using the stable version of the library.';
```

Comment:
Somewhere I read that @ericholscher used the word "unreleased" here instead and I liked.

I think it's clearer to use a sentence like,

_You are using an unreleased version of this library_

Going deeper on this (maybe in another PR focusing more on this), words like "library" and "using" don't make too much sense for all type of documentation that RTD hosts. Maybe changing them to "documentation" and "reading" are more general.

_You are reading an unreleased version of this documentation_

What do you think?

Example from Django docs,

>  This document is for Django's development version, which can be significantly different from previous releases. For older releases, use the version selector floating in the bottom right corner of this page. 

https://docs.djangoproject.com/en/dev/intro/tutorial01/

I also like the "tip" saying where you can find the other versions. I'd like to go in that direction, but I'm not sure how to write a good English sentence for this, @RichardLitt want to help us?

Reply:
> You are reading an unreleased version of this documentation

I was thinking the same, the word _documentation_ makes more sense here.

Also, we are missing some i18n on this message, I'm not sure how relevant is that for now.

About the tip, I think would be a little confusing, since the position of the version selector change according to the theme (some times is on the left and others floating to the right), and we asre already providing a link to go to the stable version.

Reply:
> Also, we are missing some i18n on this message, I'm not sure how relevant is that for now.

Good point! I think it's relevant, but I'd say to open another issue to track this. I don't know how we are handling i18n in Javascript. Did you see other parts in the code?

> About the tip, I think would be a little confusing [...]

Yeah, you are right. I think there is no _generic_ way to point the users to the right place.